src/components/WorkerPoolQueueToggle.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <p-toggle v-if="can.update.worker_pool_queue" v-model="internalValue" :state="state" />
+</template>
+
+<script lang="ts" setup>
+  import { PToggle, showToast, State } from '@prefecthq/prefect-design'
+  import { computed, reactive, toRefs } from 'vue'
+  import { useCan, useWorkspaceApi } from '@/compositions'
+  import { localization } from '@/localization'
+  import { WorkerPoolQueue } from '@/models'
+
+  const props = defineProps<{
+    workerPoolName: string,
+    workerPoolQueue: WorkerPoolQueue,
+  }>()
+
+  const { workerPoolName } = toRefs(props)
+  const { workerPoolQueue } = toRefs(props)
+
+  const emit = defineEmits<{
+    (event: 'update'): void,
+  }>()
+
+  const can = useCan()
+  const api = useWorkspaceApi()
+
+  const internalValue = computed({
+    get() {
+      return !props.workerPoolQueue.isPaused
+    },
+    set(value: boolean) {
+      toggle(value)
+    },
+  })
+
+  const state = reactive({ pending: false } as State)
```

Comment:
Didn't we remove the pending state from another toggle component because it just flashed?

Reply:
We did it in some other button, but I agree it's not needed here